### ISSUE TITLE
bug: iOS scroll bugs when "starting" from button

### ISSUE THREAD

**Issue by EinfachHans** [ionic-team/ionic-framework#22427]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**

I just mentioned a scroll Bug on iOS (mostly on real device). If i start the touch on an `ion-button` Element it appears to be very buggy:

<img src="https://user-images.githubusercontent.com/20899975/98095526-ff270700-1e8a-11eb-877c-9b9551c080ce.gif" width="300">

(first normal, following started from button)

**Expected behavior:**

To not bug when scrolling started from button

**Steps to reproduce:**

**Related code:**

Repo: https://github.com/EinfachHans/button-scroll-bug
Please ignore the `<ion-nav>` Logic, i though this caused the bug in my main app

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.12.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.4.1
   @angular-devkit/build-angular : 0.1000.8
   @angular-devkit/schematics    : 10.0.8
   @angular/cli                  : 10.0.8
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : ios 6.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 4 other plugins)

Utility:

   cordova-res (update available: 0.15.2) : 0.15.1
   native-run (update available: 1.2.2)   : 1.0.0

System:

   Android SDK Tools : 25.2.3 (/Users/hanskrywalsky/Library/Android/sdk)
   ios-deploy        : 1.10.0
   ios-sim           : 8.0.2
   NodeJS            : v12.18.3 (/usr/local/bin/node)
   npm               : 6.14.8
   OS                : macOS Catalina
   Xcode             : Xcode 12.1 Build version 12A7403
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/21989#issuecomment-681953982.

**Comment by EinfachHans** [comment]
@liamdebeasi thx - is there any workaround or do we need to wait?

**Comment by liamdebeasi** [comment]
No easy workaround at the moment other than avoiding additional paints/layouts (maybe try changing the opacity instead of the background color on focus/active?). The bug has been fixed in WebKit, it just has not shipped yet.

**Comment by EinfachHans** [comment]
@liamdebeasi I think it is fixed in 14.2 - released today 😊
